ftplugin/python/pytest.vim

```diff
@@ -896,7 +896,7 @@ function! s:ThisMethod(verbose, ...)
       return
     endif
     if ((a:1 == '--pdb') || (a:1 == '-s'))
-        call s:Pdb(path, a:1)
+        call s:Pdb(path, a:1, m_name)
```

Comment:
Yes: you would only need to check if the argument value of `parametrized_flag` is '0'. If it isn't then consume that.

No need for the `if` and `else`, just the `if (a:0 >= 2)`

I think that would be it